hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -716,18 +707,13 @@ public static void closeRegionSilentlyAndWait(ClusterConnection connection, Serv
    * @throws RetriesExhaustedException wrapping a ConnectException if failed
    */
   public AdminService.BlockingInterface getRsAdmin(final ServerName sn) throws IOException {
-    AdminService.BlockingInterface admin = this.rsAdmins.get(sn);
-    if (admin == null) {
-      LOG.debug("New admin connection to " + sn.toString());
-      if (sn.equals(master.getServerName()) && master instanceof HRegionServer) {
-        // A master is also a region server now, see HBASE-10569 for details
-        admin = ((HRegionServer) master).getRSRpcServices();
-      } else {
-        admin = this.connection.getAdmin(sn);
-      }
-      this.rsAdmins.put(sn, admin);
+    LOG.debug("New admin connection to " + sn.toString());
```

Comment:
can you change this to use a `{}`, and just pass in `sn` instead of `sn.toString()`? Not sure how useful the log is now, we could remove it maybe, but at the least we should avoid the minor bad logging practice

Reply:
nice idea. agreed it could be useless, but I want to keep this log in so that we've got some ability to investigate if the removal of caching causes more work than we've anticipated here